src/components/ClientQuote/ClientQuote.module.scss

```diff
@@ -0,0 +1,67 @@
+.main {
+  display: flex;
+  flex-direction: row;
+
+  @media (max-width: 768px) {
+    flex-direction: column-reverse;
+  }
+}
+
+.clientQuote {
+  padding: 30px 20px;
+  width: 30vw;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: lightgrey;
+  border-right: 4px solid white;
+
+  @media (max-width: 768px) {
+    width: 100vw;
+    border: none;
+  }
+}
+
+.clientQuote > p {
+  color: var(--black-font);
+  font-size: 2.2vh;
```

Comment:
id be careful with using vw or vh for font-size. it can work nicely, but related to my other comment. we should have a global css file which defines h1,h2,h3, p etc. Typically, it's better to use media queries to adjust the size. Having a variable font size could lead to unexpected results since it's adjusted for every single viewheight/width